
inline C3::Exception::Exception( const std::string& message ) :
    std::runtime_error( message )
{}
